sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -641,6 +698,7 @@ export interface FileForceCloseHandlesHeaders {
 // @public
 export interface FileForceCloseHandlesOptions extends CommonOptions {
     abortSignal?: AbortSignalLike;
+    leaseAccessConditions?: LeaseAccessConditions;
```

Comment:
Please check if the header is added correctly. Looks not change in this API version.